src/renderer/components/ft-subscribe-button/ft-subscribe-button.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <div class="buttons">
+      <ft-button
+        :label="subscribedText"
+        class="subscribeButton"
+        background-color="var(--primary-color)"
+        text-color="var(--text-with-main-color)"
+        @click="handleSubscription"
+      />
+      <ft-button
```

Comment:
An aria-label or title attribute should be added to this element for accessibility reasons